go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/coredns/coredns v1.4.0
 	github.com/coreos/bbolt v1.3.3 // indirect
 	github.com/coreos/etcd v3.3.12+incompatible
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // indirect
```

Comment:
You are re-adding a lot of un-needed packages removed by #8197

Could you check why they are added? It could be older Go version?

Reply:
Most probably because @Praveenrajmani is not yet using `go1.13`

Reply:
OOPS.. yes it was go1.12. Will update